script.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```

Comment:
- [ ] [REQUIRED] Kindly ensure that your design looks exactly like the design provided in template 2 of the figma. Let's stick to the properties e.g. font, colors, text, margins.
 ### Your Design
<img width="413" alt="Screenshot 2022-11-17 at 00 15 25" src="https://user-images.githubusercontent.com/58692590/202315473-e22dd90e-d0ad-4cc1-ab93-df5f62390385.png">

### Figma Design
<img width="381" alt="Screenshot 2022-11-17 at 00 17 33" src="https://user-images.githubusercontent.com/58692590/202315570-32593940-e9bb-41e2-8378-b7f711758bb6.png">